AWSTranscribeStreaming/Internal/AWSSRWebSocketDelegateAdaptor.m

```diff
@@ -174,9 +175,42 @@ - (void)webSocket:(AWSSRWebSocket *)webSocket
     });
 }
 
+#pragma mark - Optional protocol methods to fulfill Socket Rocket protocol
+
+//needed to fulfill Socket Rocket's protocol
+- (void)webSocket:(AWSSRWebSocket *)webSocket didReceiveMessage:(id)message {
+    // Part of wss protocol, ignore
+    [self didReceiveMessage:webSocket message: message];
+}
```

Comment:
please correct me if my understanding is incorrect here; we are fulfilling Socket Rocket's delegate (websocket didReceiveMessage) by calling didReceiveMessage:message (our delegate protocol method). i believe this is the right way to go since as the SocketRocket provider, we need to implement SRWebSocket's delegate protocol to receive data and pass that to to our delegate methods to pass that data upstream.

Upstream caller instantiates an instance of the socket rocket provider, and will get data on the defined websocket protocol and delegate protocol methods.

Reply:
yes thats exactly what I am doing, just feels icky